doc/source/dev/data_managers.rst

```diff
@@ -530,6 +536,15 @@ referenced by the Data Manager Tool, if any, is stored within the
 A data manager tool can use a ``conda`` environment if the target
 Galaxy is version 18.09 or above (specified in the tool's XML file).
 
+Data manager tools are loaded by referring to them in the data_manager_conf.xml file for the repository.
+
+Data table configuration
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Data tables that the data managers operate on are specified in files at `tool-data/*.loc.sample` relative to the repository root.
```

Comment:
The loc files are samples of how tabular data tables should look like.
The `tool_data_table_conf.xml.sample` at the repository root defines the data tables.

Reply:
Is that search using the same function as the data_manager_conf.xml?
Why is tool_data_table_conf.xml.sample suffixed with .sample but data_manager_conf.xml isnt?

Reply:
`data_manager_conf.xml` describes the data manager you've written. `tool_data_table_conf.xml.sample` is copied to shed_tool_data_path and filled with details about the new tool data table.

Reply:
So it is the tool itself that specifies the data table? And tool_data_table_conf.xml.sample is strictly a sample table config that could possibly pre-populate data tables?

Reply:
`data_manager_conf.xml` defines the tool that generates data and how to move generated data and fill the loc file programmatically.
If you want to pre-populate a data table you'd add an entries to the loc file sample.